### DISCUSSION TITLE
Downgrading from Github Enterprise to Github free for organization, what do i need to prepare?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#61759] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi everybody,

We're going to downgrade from github enterprise (using the credits from Microsoft for Startup) to github free. Do we still have access to all our private repo from the main organization page after the downgrade? How do I best prepare for this downgrade? Thank you!

Cheers!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @hnguyen1-bxtheory! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by ghost** [reply]
Thank you!

**Comment by nirav-gajera** [comment, marked as answer]
Yes, you will still have access to all your private repositories from the main organization page after you downgrade to GitHub Free. However, there are some limitations to what you can do with private repositories on GitHub Free. For example, you will only have 2,000 Actions minutes per month for private repositories, and you will not be able to use some of the advanced code review tools.

Here are some things you can do to prepare for the downgrade:

* Make sure you have a backup of all your private repositories.
* Review the limitations of GitHub Free for private repositories.
* Consider migrating any critical private repositories to a paid plan.

Here are the steps on how to downgrade from GitHub Enterprise to GitHub Free:

1. Go to your organization's settings page.
2. Click on "Billing and plans".
3. Under "Current plan", use the "Edit" drop-down and click "Downgrade to Free".
4. Read the information about the features your organization will no longer have access to on your next billing date, then click "I understand".
5. Click "Downgrade my organization".

source: https://docs.github.com/en/billing/setting-up-paid-accounts-for-procurement-companies/setting-up-paid-organizations-for-procurement-companies/upgrading-or-downgrading-your-clients-paid-organization

**Reply by ghost** [reply]
Got it, thank you very much for your response, Nirav!

**Comment by sreeshas** [comment]
im actually trying to downgrade from github enterprise to github free. i dont know how to do it. not seeing such options in UI. can someone help me?

**Comment by ohadperry** [comment]
I'm also unable to downgrade from enterprise to free account . help anyone ?

**Comment by jerryavance** [comment]
how do i downgrade to a free account? thy seem to offer no way to do this

**Comment by Shackless** [comment]
same here. @nirav-gajera please help, thank you!

**Comment by richtu4n** [comment]
Check out https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/downgrading-your-accounts-plan#downgrading-your-enterprise-accounts-plan
